ffwd-client/src/test/java/com/spotify/ffwd/v1/ValueTest.java

```diff
@@ -86,12 +88,12 @@ public Object apply(Value arg) {
 
    }
 
-  private static ByteBuffer createSample(){
-    final String fakeDistData = "FAKEDISTRIBUTIONFAKEDISTRIBUTION";
-    final byte [] bytes = fakeDistData.getBytes(StandardCharsets.UTF_8);
+  private static ByteString createSample(){
+    final String data = "FAKEDISTRIBUTIONFAKEDISTRIBUTION";
+    final byte [] bytes = data.getBytes(StandardCharsets.UTF_8);
     ByteBuffer out = ByteBuffer.allocate(bytes.length);
     out.put(bytes);
-    return out;
+    return ByteString.copyFrom(out.array());
```

Comment:
Seems unnecessary to go from byte[] -> ByteBuffer -> ByteString

Simplify to ByteString.copyFromUtf8(...)?

copyFromUtf8
--




copyFromUtf8
--